tests/extendrtests/src/rust/build.rs

```diff
@@ -36,11 +36,14 @@ fn main() {
         .split_ascii_whitespace()
         .next()
         .unwrap();
-    dbg!(raw_r_version);
     let raw_r_version: Vec<u32> = raw_r_version
         .split('.')
         .map(|x| x.parse().unwrap())
         .collect();
+    assert!(
+        raw_r_version.len() >= 3,
+        "rust version was not detected properly"
```

Comment:
I believe it is 
```suggestion
        "R version was not detected properly"
```
no?